cmd/controller/certificates.go

```diff
@@ -188,7 +188,7 @@ func (c *Certificates) Init(ctx context.Context) error {
 		"kubernetes.default",
 		"kubernetes.default.svc",
 		"kubernetes.default.svc.cluster",
-		"kubernetes.svc.cluster.local",
+		fmt.Sprintf("kubernetes.svc.%s", c.ClusterSpec.Network.ClusterDomain),
```

Comment:
This has some implicit requirements on the given `clusterDomain`, such as it cannot start with a `.`. There should probably be some validation for it.

Reply:
done using https://pkg.go.dev/github.com/asaskevich/govalidator?utm_source=godoc#IsDNSName